extensions/amp-story/1.0/story-analytics.js

```diff
@@ -28,6 +28,8 @@ export const StoryEventType = {
   PAGE_VISIBLE: 'story-page-visible',
```

Comment:
Nit: not your doing, but should we rename `StoryEventType` into something more explicit like `AnalyticsEvent`?